<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="QuotesTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>751</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Load">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_btn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>440</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Author_textBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>501</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Author</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Search_TextBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>571</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>410</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="QuoteTxtbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>501</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quote</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="GenreTxtbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>501</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Genre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TitleTxtbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>501</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SourceTxtbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>501</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Source</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SourceTypeTxtbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>501</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Source Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>iQoutes</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
